core/controllers/contributor_dashboard.py

```diff
@@ -823,34 +986,79 @@ def _get_client_side_stats(
     stats_dicts = [
         stats.to_dict() for stats in backend_stats
     ]
-    topic_ids = [
-        stats_dict['topic_id']
-        for stats_dict in stats_dicts
-    ]
+    topic_ids = []
+    for stats_dict in stats_dicts:
+        # Here 'topic_id' can never be None because stats_dict is a dictionary
```

Comment:
This won't work, remember we talked before about how assertions can't assume particular behaviour from caller functions which are currently calling the code. You'll need to do an exception, or strictly type the dict.

Reply:
An exception is raised. Thanks to point that out it was supposed to be done before the PR review, sorry.